src/coreComponents/constitutive/relativePermeability/TableRelativePermeabilityHysteresis.cpp

```diff
@@ -487,6 +500,8 @@ void TableRelativePermeabilityHysteresis::computeLandCoefficient()
   integer ipNonWetting = 0;
 
   integer const numPhases = m_phaseNames.size();
+  integer const numDir = m_drainageWettingNonWettingRelPermTableNames.size(0);
```

Comment:
in order no to have to repeat every time the capture of `numDir`, maybe make is a member variable of the class and assign and check consistency in the `postInputInitialization()`